readyapi4j-core/src/test/java/com/smartbear/readyapi4j/TestRecipeBuilderTest.java

```diff
@@ -240,6 +245,37 @@ public void buildsRecipeWithDelayTestStep() throws Exception {
         assertThat(testStep.getName(), is(testStepName));
     }
 
+    @Test
+    public void buildRecipeWithPropertiesTestStep() throws Exception {
```

Comment:
To be consistent with the other test methods, name the new ones buildsXXX rather than buildXXX.